diff --git a/apps/weblibre/lib/main.dart b/apps/weblibre/lib/main.dart
--- a/apps/weblibre/lib/main.dart
+++ b/apps/weblibre/lib/main.dart
@@ -23,7 +23,7 @@ import 'dart:convert';
 import 'dart:developer';
 
 import 'package:background_fetch/background_fetch.dart';
 import 'package:country_codes/country_codes.dart';
 import 'package:dynamic_color/dynamic_color.dart';
 import 'package:flutter/foundation.dart';
-import 'package:flutter/material.dart';
+import 'package:material_ui/material_ui.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_hooks/flutter_hooks.dart';
